storage/innobase/xtrabackup/doc/source/howtos/setting_up_replication.rst

```diff
@@ -92,7 +106,7 @@ and move the snapshot from ``TheMaster`` in its place:
 
 .. code-block:: console
 
-   TheSlave$ mv /path/to/mysql/$TIMESTAMP /path/to/mysql/datadir
+   TheSlave$ mv /path/to/mysql/backupdir /path/to/mysql/datadir
```

Comment:
Instead of `mv` it is better to use `xtrabackup --move-back --target-dir=/path/to/mysql/backupdir`. PXB will find proper datadir location from `my.cnf` and place all remote tablespaces to proper locations.